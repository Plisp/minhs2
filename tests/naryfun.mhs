main =
    let f = recfun g x y = x == y;
    in f 2;
