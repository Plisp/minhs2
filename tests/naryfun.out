main :: (Int -> Bool) = let f :: (Int -> Int -> Bool) = (recfun g :: (Int -> Int -> Bool) x y = (==) x y);
                        in f 2;
